rest_framework_extensions/mixins.py

```diff
@@ -4,6 +4,7 @@
 from rest_framework_extensions.utils import get_rest_framework_features
 from rest_framework_extensions.bulk_operations.mixins import ListDestroyModelMixin, ListUpdateModelMixin
 from rest_framework_extensions.settings import extensions_api_settings
+from rest_framework.exceptions import NotFound
```

Comment:
To support older versions of DRF (< 3.1), you probably want to use Django's `Http404` exception instead.